docs/conversion-tables/scss-lint.md

```diff
@@ -23,7 +23,7 @@ in the table.
 | --------------------------- | ----------------------- |
 | **[BangFormat][]** *enabled*<br>**`space_before_bang`**<br>`true` *default*<br>`false`<br><br>**`space_after_bang`**<br>`true`<br>`false` *default* | **[declaration-bang-space-before][] / [declaration-bang-space-after][]**<br>**`declaration-bang-space-before`**<br>`always`<br>`never`<br><br>**`declaration-bang-space-after`**<br>`always`<br>`never` |
 | **[BemDepth][]** *disabled*<br>**`max_elements`**<br>`number` *default: `1`* | - |
-| **[BorderZero][]** *enabled*<br>**`convention`**<br>`zero` *default*<br>`none` | - |
+| **[BorderZero][]** *enabled*<br>**`convention`**<br>`zero` *default*<br>`none` | **[property-value-blacklist][]**<br> `"/^border/": ["none"]`<br>`"/^border/": ["0"]` |
```

Comment:
Can you add another `<br>` after the headline to make the options align with the scss-lint options?